### PR TITLE
docs(NODE-4725): Change docs to use "in use encryption" terminology

### PR DESCRIPTION
**Pull request by nbbeeken**
Update docs NODE-4725

### CODE REVIEW

**Review by durran — APPROVED** [review]

**Review by durran — APPROVED** [review]

File: bindings/node/README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ npm test
 <strong>NOTE</strong>: Not meant to be instantiated directly, this is for internal use only.</p>
 </dd>
 <dt><a href="#ClientEncryption">ClientEncryption</a></dt>
-<dd><p>The public interface for explicit client side encryption</p>
+<dd><p>The public interface for explicit in use encryption</p>
```
Comment (dariakp) [inline bindings/node/README.md:52]:
```suggestion
<dd><p>The public interface for explicit in-use encryption</p>
```

File: bindings/node/README.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ Create an AutoEncrypter
 
 **Note**: Supplying `options.schemaMap` provides more security than relying on JSON Schemas obtained from the server.
 It protects against a malicious server advertising a false JSON Schema, which could trick the client into sending unencrypted data that should be encrypted.
-Schemas supplied in the schemaMap only apply to configuring automatic encryption for client side encryption.
+Schemas supplied in the schemaMap only apply to configuring automatic encryption for in use encryption.
```
Comment (dariakp) [inline bindings/node/README.md:146]: this is CSFLE-only

File: bindings/node/README.md [hunk 3]
```diff
@@ -239,7 +239,7 @@ the underlying C++ Bindings.
 <a name="ClientEncryption"></a>
 
 ## ClientEncryption
-The public interface for explicit client side encryption
+The public interface for explicit in use encryption
```
Comment (dariakp) [inline bindings/node/README.md:242]:
```suggestion
The public interface for explicit in-use encryption
```

File: bindings/node/index.d.ts [hunk 4]
```diff
@@ -354,7 +354,7 @@ export interface ClientEncryptionEncryptOptions {
 }
 
 /**
- * The public interface for explicit client side encrption.
+ * The public interface for explicit in use encryption
```
Comment (dariakp) [inline bindings/node/index.d.ts:357]:
```suggestion
 * The public interface for explicit in-use encryption
```

File: bindings/node/lib/autoEncrypter.js [hunk 5]
```diff
@@ -68,7 +68,7 @@ module.exports = function (modules) {
      *
      * **Note**: Supplying `options.schemaMap` provides more security than relying on JSON Schemas obtained from the server.
      * It protects against a malicious server advertising a false JSON Schema, which could trick the client into sending unencrypted data that should be encrypted.
-     * Schemas supplied in the schemaMap only apply to configuring automatic encryption for client side encryption.
+     * Schemas supplied in the schemaMap only apply to configuring automatic encryption for in use encryption.
```
Comment (dariakp) [inline bindings/node/lib/autoEncrypter.js:71]: this is csfle-only

File: bindings/node/lib/clientEncryption.js [hunk 6]
```diff
@@ -64,7 +64,7 @@ module.exports = function (modules) {
    */
 
   /**
-   * The public interface for explicit client side encryption
+   * The public interface for explicit in use encryption
```
Comment (dariakp) [inline bindings/node/lib/clientEncryption.js:67]:
```suggestion
   * The public interface for explicit in-use encryption
```

**Review by dariakp — CHANGES_REQUESTED** [review]
I think the docs are generated but I left the comments in all the lines just in case

**Review by dariakp — APPROVED** [review]

**Pull request merged**